src/static/pycontw-2020/styles/gallery.scss

```diff
@@ -0,0 +1,56 @@
+.my-gallery {
+	width: 100%;
+	float: left;
+}
+.my-gallery img {
+	width: 100%;
+	height: auto;
+}
+.my-gallery figure {
+	display: block;
+	float: left;
+	margin: 0 5px 5px 0;
+	width: 150px;
+}
+.my-gallery figcaption {
+	display: none;
+
+}
+.links{
+	text-align:center;
+	display:flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	> .frame-square{
+		flex:0 0 calc(25% - 1.5em)
+	}
+}
```

Comment:
Let's see if we remove the `justify-content: space-between;` from `.links`.
And add `margin-left: -0.5em` to `.links` and `margin-left: 0.5em` to `.links > .frame-square`.

So the code should be something like this:
```scss
.links{
	text-align:center;
	display:flex;
	flex-wrap: wrap;
	margin-left: -0.5em;
	> .frame-square{
		flex:0 0 calc(25% - 1.5em);
    margin-left: 0.5em;
	}
}
```

The trick is: we try to put 4 items in a row, so it is 25%, and we try to put 3 0.5em spaces between, which is 1.5em.
And then add the 0.5em margin for each item.
Finally, substract the 0.5em at the container so it will be centered.

Reply:
But, it looks like it won't work in mobile. Let me figure something out later...

Reply:
OK, I'm wrong, `-1.5em` in the calc is not correct. It should be `calc(25% - 14px - 0.5em)`.
The 14px comes from the 2px (border) & 5px (padding) from the `.frame-square`, and you'll need to count both sides (left & right). And the space between items is 0.5em.

If we want to do it mobile friendly, we can setup different rules for desktop & mobile.
It would be something like this
```scss
.links{
    text-align:center;
    display:flex;
    flex-wrap: wrap;
    margin-left: -0.5em;
    > .frame-square{
        flex:0 0 calc(50% - 14px - 0.5em);
        margin-left: 0.5em;

        @include on-desktop {
            flex:0 0 calc(25% - 14px - 0.5em);
        }
    }
}
```

Don't forget to `@import 'site'` at the beginning of the file.

Reply:
OK. I totally forgot something. We're using "content-box" here. I'm very used to using "border-box" since I use bootstrap all the time.

So, if we use add `box-sizing: border-box;` to `.frame-square`, we don't need the `-14px` since border and padding ARE included in size calculation. It might be easier to understand.